.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -35,10 +31,8 @@ Issue Number: N/A
 
 ## Does this PR introduce a breaking change?
 
-```
-[ ] Yes
-[ ] No
-```
+- [ ] Yes
+- [ ] No
```

Comment:
Why did we revert to real checkbox syntax? This will make pull requests appear as *[1/15] completed*.

Reply:
Oh didn't see that behavior, I reverted it because it allowed for checkbox behavior in the preview mode.